### PR TITLE
Fix and enhance the notes GPX files

### PR DESCRIPTION
**Pull request by marcows**
Related to #457.

I could not run the test suite. "bundle exec rake test" failed due to pending
migrations, but "bundle exec rake db:migrate RAILS_ENV=test" failed as well.

### CODE REVIEW

File: app/views/notes/_note.gpx.builder [hunk 1]
```diff
@@ -1,9 +1,14 @@
 xml.wpt("lon" => note.lon, "lat" => note.lat) do
+  xml.time note.created_at.to_s(:iso8601)
+  xml.name "#{t'note.mine.description'} #{note.id}"
```
Comment (tomhughes) [inline app/views/notes/_note.gpx.builder]: I don't understand the use of `note.mine.description` here? That gives a description of "Description 42" for note 42 which seems very odd...

Comment (tomhughes) [inline app/views/notes/_note.gpx.builder]: I suspect the `#{t'browse.note.title', :id => note.id}` which gives "Note: 42" would work much better?

Comment (marcows) [inline app/views/notes/_note.gpx.builder]: You're right.
Sorry, I did not look properly for the suitable translation. I searched for "Hinweis" in the German translations, which is the proper translation for "note", but unfortunately the translation does not go the direct way in the user notes page ("description" normally is just "Beschreibung"), which led to this bad choice.

I'll update the PR.

**Comment by marcows** [comment]
@tomhughes I have updated the PR again and use `browse_note_url()` for getting the URL, as noted in my previous comment (which disappeared after the force push). Is it ok now?

**Comment by tomhughes** [comment]
Merged as 166dc76ae94de4b07b32836e6db938e4040eaf09.

**Pull request closed without merging**